Allow embeddable users to use the new image options of the editor (#271)
lblod/frontend-embeddable-notule-editor: docs/configuration.md

- `docContent: 'block+'`: The property docContent specifies which nodes are allowed in the document. By default we allow one or more nodes of the group block, which includes most content. A group can be seen as a supertype that includes multiple types. For more info about this check the [Prosemirror docs](https://prosemirror.net/docs/guide/#schema.content_expressions).
  See `public/test.html` where `docContent` is specified to allow a [table of contents](/docs/plugins/table-of-contents-plugin.md) and [article-structure](/docs/plugins/article-structure-plugin.md) nodes in a specific order.
- `ui: { expandInsertMenu: false }`: Whether to automatically open the "insert" sidebar menu upon load.
- `table: { inlineBorderStyle: { width: '1px', style: 'solid', color: '#000' }`: Styles to be applied to table borders in the editor and when exporting HTML (e.g. by copying to the clipboard). If not specified, a thin border is shown in the editor but not exported.
- `table: { rowBackgrounds: { even: 'white', odd: 'whitesmoke' } }`: Configuration of row background colors. Allows for specifying alternating row colors. Rows use 1-based indexing.

> - `image: { allowBase64Images: true}`: Will enable the pasting of base64 images embedded into the document, also enables a plugin to check if the paste size is too big. By default the paste limit will be 100kb and will throw an error to the console if surpased.

Reviewer: ```suggestion
- `image: { allowBase64Images: true}`: Will enable the pasting of base64 images embedded into the document, also enables a plugin to check if the paste size is too big. By default the paste limit will be 100kb and will throw an error to the console if surpassed.
```

> - `image: { allowBase64Images: true, pasteLimit: 2000000, onLimitReached: () => console.error('You can only paste up to 2 MB')}`: You have 2 options to configure the paste size limit plugin, first the pasteLimit which will determine in bytes the limit of size the user can

Reviewer: ```suggestion
- `image: { allowBase64Images: true, pasteLimit: 2000000, onLimitReached: () => console.error('You can only paste up to 2 MB')}`: You have 2 options to configure the paste size limit plugin, first the pasteLimit which will determine the size limit in bytes the user can
```

>   paste, if this limit is surpased the onLimitReached function will be called, which allows to show appropiate feeback to the user of your app.

Reviewer: ```suggestion
  paste, if this limit is surpassed the onLimitReached function will be called, which allows to show appropriate feedback to the user of your app.
```
